content/docs/1.8.0/deploy/install/_index.md

```diff
@@ -158,6 +158,7 @@ Below are the directories Longhorn components requiring access with root and pri
 - Longhorn Manager
   - /dev: Block devices created by Longhorn are under the `/dev` path.
   - /proc: Find the recognized host process like container runtime, then use `nsenter` to access the mounts on the host to understand disks usage.
+  - /etc: Read the necessary system configuration to get node status updated, e.g. `nfsmount.conf`.
```

Comment:
For your consideration. Prefer English rather than Latin exempli gratia. Thanks
```suggestion
  - /etc: Read the necessary system configuration to get node status updated, for example, `nfsmount.conf`.
```